### ISSUE TITLE
no way to move an imported picture after one try

### ISSUE THREAD

**Issue by MatthieuS-ctrl** [musescore/MuseScore#24791]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

I tried with .jpg and .png with small files of around 150KB, which I have always used to illustrate scores for children.

The image can be moved for the first time, and you can continue to move it as long as it is not deselected.

Once deselected, the number of test shots becomes random: either it is impossible to move it again (the image is then integrated into the score, and it is the whole that moves), or you are entitled to do so. to a second manipulation possibility, but I was able to go up to 4.

In short, I haven't been able to precisely identify the problem....

### Supporting files, videos and screenshots

![459440959_3859847137578202_8334505033483737385_n](https://github.com/user-attachments/assets/0d7d8634-90f6-47d9-82df-87da1df2e566)

### What is the latest version of MuseScore Studio where this issue is present?

4.4.2

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 10

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by oktophonie** [comment]
Can you let us know which version of MuseScore you are using? I cannot reproduce this in 4.4.2 in Linux or Windows.

**Comment by MatthieuS-ctrl** [comment]
Hello, it happened on version 4.4.1, so I update with 4.4.2 yesterday, and there is still this problem

**Comment by MatthieuS-ctrl** [comment]
I tried with .jpg and .png with small files of around 150KB, which I have always used to illustrate scores for children.

The image can be moved for the first time, and you can continue to move it as long as it is not deselected.

Once deselected, the number of test shots becomes random: either it is impossible to move it again (the image is then integrated into the score, and it is the whole that moves), or you are entitled to do so. to a second manipulation possibility, but I was able to go up to 4.

In short, I haven't been able to precisely identify the problem....

**Comment by MarcSabatella** [comment]
I'm not able to reproduce any problem with this, but I wonder if you're getting confused by the way autoplace adjusts things to avoid collisions as you move?  Or perhaps you have other elements in your score with a higher stacking order (see Appearance section of Properties panel) than the image, so they overlap the image and make it harder to select?

If you continue to have problems after verifying it isn't just one of those things, try making a screen recording and uploading that here.

**Comment by cadiz2** [comment]
I can confirm that this fix does not resolve the issue satisfactorily. New created default flute score (so nothing to deal with autoplace, stacking order),  see what happens on the GIF below.
 Moved once with the mouse, it then sticks in place and causes the score to move. In short, this fix needs improvement.

![Video 2](https://github.com/user-attachments/assets/4038107d-62b5-49fc-a42a-3e0203cb5b3b)

**Comment by MarcSabatella** [comment]
Linux nightlies seems broken or disabled?  No 4.4.3 build, no other updates since the 17th.

**Comment by zacjansheski** [comment]
@MatthieuS-ctrl  I was asking if it is fixed in nightly for you (it seems much better for me) https://musescore.org/en/nightly-builds

@MarcSabatella Linux Nightly builds are fixed now by https://github.com/musescore/MuseScore/pull/24825

**Comment by MatthieuS-ctrl** [comment]
@zacjansheski sorry my english is not very good .....
 
 I've installed this file:
 
MuseScoreNightly-latest-master-x86_64-portable.exe | 2024-09-20 03:31 | 95M |  

which is version

OS: Windows 10 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.5.0-242640305, revision: github-musescore-musescore-1aec102

Imported picture is stuck after one click. Then all the score is moving.

I click on the score, and on the picture again, so I can move the picture one more time.

After that more try, no way to move the picture again.

**Comment by bkunda** [comment]
Just a bit of extra info: I was able to reproduce this, but then I could re-select my picture if I made a change to the score layout (e.g. by inserting a system break). But it quickly broke again, particularly after clicking and dragging the score itself.
